<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientAnalyseItem</class>
 <widget class="QWidget" name="PatientAnalyseItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblIcon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblType">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>178</width>
     <height>194</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(70, 86, 134);  </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>btnOpen</zorder>
  <zorder>lblIcon</zorder>
  <zorder>lblName</zorder>
  <zorder>lblType</zorder>
  <zorder>lblDate</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
